### DISCUSSION TITLE
Reimport Blueprint at HA 23.12

### DISCUSSION THREAD

**Discussion by Churator** [Blackymas/NSPanel_HA_Blueprint#1396] [General]
Finally :) one easier step to upgrade NSPANEL :)
https://www.home-assistant.io/blog/2023/12/06/release-202312/#re-import-blueprints:~:text=very%20helpful%20improvement.-,Re%2Dimport%20blueprints,-This%20release%20adds

**Comment by edwardtfn** [comment]
Yeah.
The update instructions are updated. 😉
